android/filament-android/src/main/java/com/google/android/filament/View.java

```diff
@@ -160,6 +160,14 @@ public AntiAliasing getAntiAliasing() {
         return AntiAliasing.values()[nGetAntiAliasing(getNativeObject())];
     }
 
+    public void setDithering(boolean dithering) {
+        nSetDithering(getNativeObject(), dithering);
+    }
+
+    public boolean getDithering() {
```

Comment:
setDitheringEnabled/isDitheringEnabled, or use an enum like we do for AA (which would be better this way we could later offer different types of dithering, including non temporal)